client/layout/style.scss

```diff
@@ -168,6 +168,13 @@ body.is-mobile-app-view {
 		display: none;
 	}
 }
+
+.is-section-themes.is-global-sidebar-visible {
+	.layout__content {
+		padding: calc(var(--masterbar-height) + 16px) 16px 16px calc(var(--sidebar-width-max));
+		min-height: 100vh;
+	}
+}
```

Comment:
It seems like we could consolidate this into the other scss file for the themes section rules.

Reply:
Moved this - good spot!